src/frontend/src/lib/styles/global/gix.scss

```diff
@@ -145,7 +145,12 @@ div.modal {
 			border-radius: 0;
 
 			div.content {
+				height: 100vh;
```

Comment:
Sorry did not noticed it before, that's correct?

1. The inner content is not 100vh no?
2. On desktop we don't want 100vh?